intents-operator/values.yaml

```diff
@@ -46,4 +46,7 @@ allowGetAllResources: true
 watchedNamespaces: null # by default, watch all
 global:
   # If defined overrides `allowGetAllResources`
-  allowGetAllResources:
\ No newline at end of file
+  allowGetAllResources:
+  clientID:
```

Comment:
what do you think about global.otterize.apiAddress and global.otterize.clietnt* won't it be prettier?